tests/utils_test.py

```diff
@@ -93,7 +93,7 @@ def testConstructTechnicalDetail_whenMissingSomeDetail_returnsTechnicalDetail()
     """Unittest for the technical detail generation:
     case when all details are provided
     """
-    vulnerability_json = VULNERABILITIES[0]
+    vulnerability_json = VULNERABILITIES[0].copy()
```

Comment:
Why the copy?

Reply:
because right after it we use `del vulnerability_json["check_id"]` to simulate a case where `check_id` is missing, if don't use a copy the `check_id` will be missing for all the next tests

Reply:
:) the fix is to make it a fixture. It should have already been a fixture.